priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.TestRunner do
+  @moduledoc """
+  This module runs tests using `mix run`. For example:
+
+  mix run -e "LoadTest.TestRunner.run()" -- "deposits" "1" "5"
+
+  It accepts three arguments:
+  - test name
+  - transactions per seconds
+  - period in seconds
+
+  It fetches all configuration params from env vars.
+  """
+  alias LoadTest.TestRunner.Config
+
+  def run() do
+    {runner_module, config} = Config.parse()
+    result = Chaperon.run_load_test(runner_module, print_results: true, config: config)
+
+    case result.metrics["success_rate"][:mean] do
+      1.0 ->
+        System.halt(0)
+
+      _ ->
+        System.halt(1)
+    end
```

Comment:
what is this?

Reply:
it's Unix exit codes:
- 0 - success
- 1 - failure

Reply:
https://tenor.com/vzXt.gif

Reply:
my question was... what are you measuring with `result.metrics["success_rate"][:mean]`? Why `mean`? how are you making sure you won't get too strict tests in performance tests?

like, you might be simply having network issues in a 72h test and you hit a 90% success rate.

Reply:
Ok. I can make this configurable
I think @boolafish  has the same comment
https://github.com/omgnetwork/elixir-omg/pull/1745/files#r503739580

Reply:
I added percentile as an optional fourth parameter https://github.com/omgnetwork/elixir-omg/pull/1745